spec/commands/decidim/action_delegator/admin/create_setting_spec.rb

```diff
@@ -38,4 +39,26 @@
       expect { subject.call }.not_to(change { Decidim::ActionDelegator::Setting.count })
     end
   end
+
+  context "when copy setting" do
+    let!(:copy_from_setting) { create(:setting, :with_participants, :with_ponderations, consultation: other_consultation) }
+    let(:other_consultation) { create(:consultation) }
+    let(:form) do
+      double(
+        invalid?: invalid,
+        max_grants: max_grants,
+        authorization_method: authorization_method,
+        decidim_consultation_id: decidim_consultation_id,
+        source_consultation_id: copy_from_setting.id
```

Comment:
```suggestion
        copy_from_setting_id: copy_from_setting.id
```
There is something wrong here as this has been renamed. Test should fail if we were testing the if the participants where copied.

Can you try to add a test case where to check that participants have been copied?

Reply:
Yes, you are right. I changed it and added new cases.